src/Sylius/Bundle/ThemeBundle/.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+
+before_script: composer install --prefer-source --no-interaction
```

Comment:
I think we should adjust it to match other .travis.yml files - https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ApiBundle/.travis.yml#L8

(we can skip the memory if not needed)